src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (a) => {
+    counter++;
+
+    return (b) => {
+      if (counter <= 3 || counter % 2 !== 0) {
+        return a + b;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
you'd rather inverse your if-statement
```
if (...) { // if something goes wrong
	// we stop doing what we're doing
}

// otherwise – we do what we need
```
this pattern looks more appropriate